pkg/plugins/golang/v4/scaffolds/internal/templates/api/webhook.go

```diff
@@ -112,13 +116,15 @@ func (r *{{ .Resource.Kind }}) SetupWebhookWithManager(mgr ctrl.Manager) error {
 	defaultingWebhookTemplate = `
 //+kubebuilder:webhook:{{ if ne .Resource.Webhooks.WebhookVersion "v1" }}webhookVersions={{"{"}}{{ .Resource.Webhooks.WebhookVersion }}{{"}"}},{{ end }}path=/mutate-{{ .QualifiedGroupWithDash }}-{{ .Resource.Version }}-{{ lower .Resource.Kind }},mutating=true,failurePolicy=fail,sideEffects=None,groups={{ .Resource.QualifiedGroup }},resources={{ .Resource.Plural }},verbs=create;update,versions={{ .Resource.Version }},name=m{{ lower .Resource.Kind }}.kb.io,admissionReviewVersions={{ .AdmissionReviewVersions }}
 
-var _ webhook.Defaulter = &{{ .Resource.Kind }}{}
+var _ admission.CustomDefaulter = &{{ .Resource.Kind }}{}
 
-// Default implements webhook.Defaulter so a webhook will be registered for the type
-func (r *{{ .Resource.Kind }}) Default() {
+// Default implements admission.CustomDefaulter so a webhook will be registered for the type
```

Comment:
Are we calling `admission.CustomDefaulter` for default webhooks? 
Or did I misunderstood? Should not it only be used for admission ones?

Reply:
`webhook.CustomDefaulter` is an alias for `admission.CustomDefaulter`. I missed that when I first looked at the required changes, but have now reverted it to use `webhook.CustomDefaulter`. I hope this answers your question.